docker/rhel.Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2018-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+# NOTE: using registry.access.redhat.com/rhel8/go-toolset does not work (user is requested to use registry.redhat.io)
+# NOTE: using registry.redhat.io/rhel8/go-toolset requires login, which complicates automation
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/devtools/go-toolset-rhel7
+FROM devtools/go-toolset-rhel7:1.12.12-4  as builder
+ENV PATH=/opt/rh/go-toolset-1.12/root/usr/bin:$PATH \
+    GOPATH=/go/
+USER root
+WORKDIR /go/src/github.com/redhat-developer/kubernetes-image-puller/
```

Comment:
is this going to be the right `WORKDIR`? This is upstream right?

Reply:
good catch -- that's the OTHER (rh-che specific) upstream.